build/patch-charts.sh

```diff
@@ -243,6 +243,14 @@ function patchGateways() {
   # do not expose port 15012, support for mesh expansion will automatically add this port
   sed_wrap -i -e '/port: 15012/,+3d' ${HELM_DIR}/gateways/istio-ingress/values.yaml
 
+  # add route config
+  routeConfig='\
+\
+    # specifies whether to create a Route resource (route.openshift.io/v1) for the istio-ingressgateway deployment\
+    routeConfig:\
+      enabled: true'
+  sed_wrap -i -e "/runAsRoot: false/a $routeConfig" ${HELM_DIR}/gateways/istio-ingress/values.yaml
+
```

Comment:
Once this [pull request](https://github.com/maistra/istio/pull/469) is merged, we can revert this patch.